### PR TITLE
Fix tests after new UI kit version merged

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix tests after new UI kit version merged
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | All tests should be OK

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21854)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

File: tests/UI/campaigns/functional/BO/03_catalog/01_products/01_filterAndQuickEditProducts.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe('Filter Products', async () => {
     ];
 
     tests.forEach((test) => {
-      filterValue = test.args.filterValue.min === undefined ? `'${test.args.filterValue}`
+      filterValue = test.args.filterValue.min === undefined ? `'${test.args.filterValue}'`
```
Comment (SimonGrn) [inline tests/UI/campaigns/functional/BO/03_catalog/01_products/01_filterAndQuickEditProducts.js:135]: That didn't break before ?

File: tests/UI/pages/BO/catalog/products/index.js [hunk 2]
```diff
@@ -183,10 +183,16 @@ class Product extends BOBasePage {
...
    */
   async getProductStatusFromList(page, row) {
-    return this.getTextContent(page, this.productsListTableColumnStatus(row));
+    const inputValue = await this.getAttributeContent(
+      page,
+      `${this.productsListTableColumnStatusInput(row)}[checked]`,
+      'value',
+    );
+
+    return inputValue !== '0';
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/products/index.js:195]: This was a string before and now it's a boolean, all the tests are fixed ?

File: tests/UI/campaigns/functional/BO/03_catalog/01_products/01_filterAndQuickEditProducts.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/03_catalog/01_products/01_filterAndQuickEditProducts.js:135]: just something to display as Scenario name, will not break anything

File: tests/UI/pages/BO/catalog/products/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/products/index.js:195]: Only one tests used the function, and it's fixed

**Pull request merged**